src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -53,7 +53,7 @@ export interface OfferState {
 
 const logger = createLogger("Order/Routes/Offer/index.tsx")
 
-const enableOfferNote = sd.ENABLE_OFFER_NOTE
+const enableOfferNote = process.env.ENABLE_OFFER_NOTE || sd.ENABLE_OFFER_NOTE
```

Comment:
can this be attached to `sd`?

Reply:
@damassi let me verify-- I added this to solve a rehydration issue but it might not be needed anymore.

Reply:
Yeah, looks like I need this so it'll render the note field on the server (otherwise it rehydrates incorrectly).

Reply:
We should really try to not need 2 ways, perhaps something needs to be done to inject the value into `sd` on the Force side?

Reply:
Ah, you are probably right! I thought there might be some outstanding issues with how we treat env vars here so the weirdness seemed OK. Ill check